etc/checkstyle.xml

```diff
@@ -100,14 +100,15 @@ under the License.
          <property name="severity" value="ignore"/>
       </module>
 
-      <!-- Sevntu checks, http://sevntu-checkstyle.github.io/sevntu.checkstyle/ -->
-      <module name="DiamondOperatorForVariableDefinition"/>
       <module name="RequiredParameterForAnnotation">
          <property name="annotationName" value="Parameterized.Parameters"/>
          <property name="requiredParameters" value="name"/>
       </module>
 
       <module name="UnnecessarySemicolonInTryWithResources"/>
 
+      <!-- Sevntu checks, http://sevntu-checkstyle.github.io/sevntu.checkstyle/ -->
+      <module name="DiamondOperatorForVariableDefinition"/>
```

Comment:
The comment seems potentially out of place now since e.g RequiredParameterForAnnotation also looks to be a Sevntu check? Did this need moved?

Reply:
I looked in the Sevntu Git repo for `RequiredParameterForAnnotation` and I couldn't find it, but I see now that I was looking in the wrong place (i.e. `coding` package instead of `annotation`). My goal here was to clarify which specific checks were from Sevntu.